File:error_log.txt:2024-03-07 10:55:49.752642 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-07 10:56:13.174387 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-07 10:56:59.482981 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 16:08:36.462950 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 16:09:29.516313 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 16:09:39.522738 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:15:47.304172 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:17:05.405936 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:22:49.057656 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:23:03.939942 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:23:31.976709 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:24:43.602429 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:25:45.004961 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:26:25.387502 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:28:37.338498 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: '/Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:28:47.343221 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:32:55.710105 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:55:38.403891 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:56:42.461631 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:59:28.031179 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: './Data/stations_raw.json'
File:error_log.txt:2024-03-11 17:59:43.627431 
Error: json_to_list() function error - FNF: [Errno 2] No such file or directory: 'Data/stations_raw.json'
2024-03-11 18:33:30.846921 - Error: extract_availability() function error - SQLAlchemy Error on entry = {'number': 42, 'contract_name': 'dublin', 'name': 'SMITHFIELD NORTH', 'address': 'Smithfield North', 'position': {'lat': 53.349562, 'lng': -6.278198}, 'banking': False, 'bonus': False, 'bike_stands': 30, 'available_bike_stands': 8, 'available_bikes': 21, 'status': 'OPEN', 'last_update': 1710181964000}: (pymysql.err.IntegrityError) (1452, 'Cannot add or update a child row: a foreign key constraint fails (`DBProject_InitName`.`availability`, CONSTRAINT `availability_ibfk_1` FOREIGN KEY (`number`) REFERENCES `station` (`number`))')
[SQL: INSERT INTO availability (number, last_update, available_bikes, available_bike_stands, status, scrape_time) VALUES (%(number)s, %(last_update)s, %(available_bikes)s, %(available_bike_stands)s, %(status)s, %(scrape_time)s)]
[parameters: {'number': 42, 'last_update': 1710181964000, 'available_bikes': 21, 'available_bike_stands': 8, 'status': 'OPEN', 'scrape_time': 1710182010}]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2024-03-11 18:34:35.864068 - Error: extract_availability() function error - SQLAlchemy Error on entry = {'number': 42, 'contract_name': 'dublin', 'name': 'SMITHFIELD NORTH', 'address': 'Smithfield North', 'position': {'lat': 53.349562, 'lng': -6.278198}, 'banking': False, 'bonus': False, 'bike_stands': 30, 'available_bike_stands': 7, 'available_bikes': 22, 'status': 'OPEN', 'last_update': 1710181986000}: (pymysql.err.IntegrityError) (1452, 'Cannot add or update a child row: a foreign key constraint fails (`DBProject_InitName`.`availability`, CONSTRAINT `availability_ibfk_1` FOREIGN KEY (`number`) REFERENCES `station` (`number`))')
[SQL: INSERT INTO availability (number, last_update, available_bikes, available_bike_stands, status, scrape_time) VALUES (%(number)s, %(last_update)s, %(available_bikes)s, %(available_bike_stands)s, %(status)s, %(scrape_time)s)]
[parameters: {'number': 42, 'last_update': 1710181986000, 'available_bikes': 22, 'available_bike_stands': 7, 'status': 'OPEN', 'scrape_time': 1710182075}]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2024-03-11 18:36:35.658486 - Error: extract_availability() function error - SQLAlchemy Error on entry = {'number': 42, 'contract_name': 'dublin', 'name': 'SMITHFIELD NORTH', 'address': 'Smithfield North', 'position': {'lat': 53.349562, 'lng': -6.278198}, 'banking': False, 'bonus': False, 'bike_stands': 30, 'available_bike_stands': 7, 'available_bikes': 22, 'status': 'OPEN', 'last_update': 1710181986000}: (pymysql.err.IntegrityError) (1452, 'Cannot add or update a child row: a foreign key constraint fails (`DBProject_InitName`.`availability`, CONSTRAINT `availability_ibfk_1` FOREIGN KEY (`number`) REFERENCES `station` (`number`))')
[SQL: INSERT INTO availability (number, last_update, available_bikes, available_bike_stands, status, scrape_time) VALUES (%(number)s, %(last_update)s, %(available_bikes)s, %(available_bike_stands)s, %(status)s, %(scrape_time)s)]
[parameters: {'number': 42, 'last_update': 1710181986000, 'available_bikes': 22, 'available_bike_stands': 7, 'status': 'OPEN', 'scrape_time': 1710182195}]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
